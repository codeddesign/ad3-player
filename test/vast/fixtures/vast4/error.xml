<?xml version="1.0" encoding="UTF-8"?>
<VAST version="4.0"><Error><![CDATA[http://adserver.com/noad.gif]]></Error></VAST>
