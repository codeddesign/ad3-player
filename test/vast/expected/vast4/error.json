{
    "$version": "4.0",
    "$error": [
        "http://adserver.com/noad.gif"
    ],
    "$ads": []
}
